dpgen2/op/run_nvnmd.py

```
@@ -0,0 +1,484 @@
+            ext = os.path.splitext(teacher_model.file_name)[-1]
+            teacher_model_file = "teacher_model" + ext
+            teacher_model.save_as_file(teacher_model_file)
+            model_files = [Path(teacher_model_file).resolve()] + model_files
+
+        with set_directory(work_dir):
+            # link input files
+            for ii in input_files:
+                iname = ii.name
+                try:
+                    Path(iname).symlink_to(ii)
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)